<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditListing</class>
 <widget class="QMainWindow" name="EditListing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Current Listing</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <activeon>ChurFindsLogo.png</activeon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #276E91; /* Replace with your desired blue hex code */
}

QWidget#centralwidget {
    /* Sets the color for the main content area */
    background-color: #276E91; 
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="ListingNameInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>255</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 1px solid gray; 
background-color: white;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="CompanyName">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>1118</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #FFFFFF; /* Sets the text color to the bright logo blue */
    font-size: 36pt; /* You can also change size here */
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;Chur Finds&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="FormTitle">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Edit Current Listing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditListingButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>620</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#EditListingButton:hover {
background-color: #276E91;
}

QPushButton#EditListingButton {
/* Keeping Search Button style the same as before for contrast */
background-color: rgb(80, 165, 255);
color: white;
border-radius: 8px;
padding: 10px 15px;
border: 2px solid #41B8F2;
font-weight: bold;
font-size: 8pt;
}</string>
    </property>
    <property name="text">
     <string>Edit Listing</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="listingDescriptionLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="listingSNameLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>465</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Seller Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="listingNameLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Listing Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ListingPriceInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>425</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 1px solid gray; 
background-color: white;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="listingSPhoneLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>465</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Seller Phone</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ListingSNameInput">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>510</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 1px solid gray; 
background-color: white;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>620</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#CancelButton {
/* Keeping Search Button style the same as before for contrast */
background-color: #FF0033;
color: white;
border-radius: 8px;
padding: 10px 15px;
border: 2px solid #C20027;
font-weight: bold;
font-size: 8pt;
}

QPushButton#CancelButton:hover {
background-color: #C20027;
}

QPushButton#CancelButton {
/* Keeping Search Button style the same as before for contrast */
background-color: #FF0033;
color: white;
border-radius: 8px;
padding: 10px 15px;
border: 2px solid rgb(149, 0, 0);
font-weight: bold;
font-size: 8pt;
}
</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>14</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="ListingSEmailInput">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 1px solid gray; 
background-color: white;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="listingSEmailLabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>465</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Seller Email</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ConditionOptions">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 1px solid gray; 
background-color: white;</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="CompanyLogo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>ChurFindsLogo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="ListingSPhoneInput">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>510</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 1px solid gray; 
background-color: white;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="ListingDescInput">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>255</y>
      <width>531</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
border: 1px solid gray; 
background-color: white;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>199</y>
      <width>890</width>
      <height>390</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
        border-radius: 15px;
        border: 1px solid #rgb(39, 110, 144); /* Silver/light gray border */
        background-color: rgb(45, 128, 167);
        padding: 5px;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="listingConditionLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>295</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Listing Condition</string>
    </property>
   </widget>
   <widget class="QLabel" name="listingPriceLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Listing Price</string>
    </property>
   </widget>
   <zorder>listWidget</zorder>
   <zorder>ListingNameInput</zorder>
   <zorder>CompanyName</zorder>
   <zorder>FormTitle</zorder>
   <zorder>EditListingButton</zorder>
   <zorder>listingDescriptionLabel</zorder>
   <zorder>listingSNameLabel</zorder>
   <zorder>listingNameLabel</zorder>
   <zorder>ListingPriceInput</zorder>
   <zorder>listingSPhoneLabel</zorder>
   <zorder>ListingSNameInput</zorder>
   <zorder>CancelButton</zorder>
   <zorder>ListingSEmailInput</zorder>
   <zorder>listingSEmailLabel</zorder>
   <zorder>ConditionOptions</zorder>
   <zorder>CompanyLogo</zorder>
   <zorder>ListingSPhoneInput</zorder>
   <zorder>ListingDescInput</zorder>
   <zorder>listingConditionLabel</zorder>
   <zorder>listingPriceLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
